src/views/ScaleSlider.js

```diff
@@ -0,0 +1,111 @@
+const BoneView = require("backbone-viewj");
+const _ = require("underscore");
+const $ = require("jbone");
+//const Slider = require("bootstrap-slider");
+
+const View = BoneView.extend({
+
+  
+  initialize: function(data) {
+    this.g = data.g;
+    this.listenTo(this.g.zoomer,"change:columnWidth", this.render);
+    this.toggleClass = 'msa-hide';
+    this.isVisible = true;
+    return this;
+  },
+  
+  attributes: {
+    class: "biojs_msa_scale"
+  },
+  
+  events: {
+    'change input': 'updateSlider',
+    'click button.msa-btn-close': 'hide',
+    'click button.msa-btn-open': 'show',
+    'click button[data-action]': 'clickButton',
+  },
+  
+  template: _.template('\
+<div class="msa-scale-minimised">\
+  <button class="btn msa-btn msa-btn-open">Zoom</button>\
+</div>\
+<div class="msa-scale-maximised">\
+  <button class="btn msa-btn msa-btn-close" style="float:right">&times; close</button>\
+  <div>\
+  <input type="range" \
+    data-provide="slider" \
+    min="<%= min %>" \
+    max="<%= max %>" \
+    step="<%= step %>" \
+    value="<%= value %>" \
+  >\
+  </div>\
+  <div class="btngroup msa-btngroup">\
+    <button class="btn msa-btn" data-action="smaller"><span class="glyphicon-zoom-out"></span>-</button>\
+    <button class="btn msa-btn" data-action="bigger"><span class="glyphicon-zoom-in"></span>+</button>\
+    <button class="btn msa-btn" data-action="reset"><span class="glyphicon-repeat"></span>reset</button>\
+  </div>\
+</div>\
+'),
+  
+  render: function() {
+    var sizeRange = this.model.getSizeRange();
+    var stash = {
+      value: this.model.getSize(),
+      min: sizeRange[0],
+      max: sizeRange[1],
+      step: this.model.step || 1,
+      colWidth: this.model.getColumnWidth(),
+    };
+    this.$el.html( this.template(stash) );
+    if ( this.isVisible ) {
+      this.show();
+    }
+    else {
+      this.hide();
+    }
```

Comment:
Other components currently are hidden by their parent, which was thought to be more flexible, but turned out to be complex and rather hard to maintain.
The advantage of this approach is that's a lot simpler, but on the other side we put something on the DOM that doesn't need to be there.

I like this approach more, but I opened #145 to find a more general approach ;-)